Rubberduck.Core/AutoComplete/AutoCompleteBase.cs

```diff
@@ -36,7 +36,16 @@ public virtual bool Execute(AutoCompleteEventArgs e, AutoCompleteSettings settin
                 {
                     var code = original.Insert(Math.Max(0, selection.StartColumn - 1), InputToken + OutputToken);
                     module.ReplaceLine(selection.StartLine, code);
-                    pane.Selection = new Selection(selection.StartLine, selection.StartColumn + 1);
+                    var newCode = module.GetLines(selection);
+                    if (newCode == code)
+                    {
+                        pane.Selection = new Selection(selection.StartLine, selection.StartColumn + 1);
+                    }
+                    else
+                    {
+                        // VBE added a space; need to compensate:
+                        pane.Selection = new Selection(selection.StartLine, selection.StartColumn + 2);
```

Comment:
The comment does not match what the code does. We are blindly assuming that a space was added simply because `code` isn't equal to `newCode`. I would wrap this in a `else if(newCode == string.Concat(code, " ")` to avoid false positive or whatever.

Reply:
@bclothier indeed it *is* assuming the VBE only added 1 space. But it's not blind: there's only 1 reason for the inserted line to mismatch the string that was inserted, and that reason is the VBE adding a space *somewhere*. Working out exactly what the diff is would be overkill IMO.